.github/workflows/auto-approve.yml

```diff
@@ -0,0 +1,13 @@
+# Approve PRs with "pr/auto-approve". mergify takes care of the actual merge.
+
+name: auto-approve
+on: pull_request
+
+jobs:
+  auto-approve:
+    if: contains(github.event.pull_request.labels.*.name, 'pr/auto-approve')
```

Comment:
Just merging PRs based on a label seems insufficient from a security standpoint. 

Maybe also have a whitelisted set of author accounts (i.e., our known bots)

Reply:
What's the security concern? Applying labels can only be done by administrators.

Reply:
Today, administrators cannot approve their own change. This opens that door.

Feels safer to have an allowlist here?

Reply:
Got it. Okay, I'll add dependabot and cdl-automation in the allow list.

Reply:
Okay check out the latest revision please.